src/main/java/com/js/tictactoe/language/Language.java

```diff
@@ -23,7 +23,11 @@ public String loadString(String string) {
       setUpReaders();
       langProperties.load(new InputStreamReader(input, Charset.forName("UTF-8")));
     } catch (NullPointerException | IOException e) {
-      logger.log(Level.INFO, e.getCause() + e.getLocalizedMessage());
+      try {
```

Comment:
http://www.myiconart.com/2114-thickbox/predator-hoodie-sweat-face-what-the-hell-are-you.jpg

Reply:
fixed